zetaclient/chains/ton/observer/outbound.go

```diff
@@ -71,7 +71,7 @@ func (ob *Observer) ProcessOutboundTrackers(ctx context.Context) error {
 		if err != nil {
 			ob.Logger().Outbound.
 				Error().Err(err).
-				Uint64("outbound.nonce", nonce).
+				Uint64("outbound_nonce", nonce).
```

Comment:
we could reuse `logs.FieldNonce` and `logs.Tx`, same for other logs in Ton observer/signer.